Apps/Playground/CMakeLists.txt

```diff
@@ -1,15 +1,17 @@
 set(BABYLONSCRIPTS
-    "../BabylonScripts/babylonjs.loaders.js"
-    "../BabylonScripts/babylonjs.loaders.js.map"
-    "../BabylonScripts/babylon.max.js"
-    "../BabylonScripts/babylon.max.js.map"
-    "../BabylonScripts/babylonjs.materials.js"
-    "../BabylonScripts/babylonjs.materials.js.map"
-    "../BabylonScripts/babylon.gui.js"
-    "../BabylonScripts/babylon.gui.js.map"
-    "../BabylonScripts/ammo.js"
-    "../BabylonScripts/meshwriter.min.js"
-    "../BabylonScripts/recast.js")
+    "../node_modules/babylonjs-loaders/babylonjs.loaders.js"
+    "../node_modules/babylonjs-loaders/babylonjs.loaders.js.map"
+    "../node_modules/babylonjs/babylon.max.js"
+    "../node_modules/babylonjs/babylon.max.js.map"
+    "../node_modules/babylonjs-materials/babylonjs.materials.js"
+    "../node_modules/babylonjs-materials/babylonjs.materials.js.map"
+    "../node_modules/babylonjs-gui/babylon.gui.js"
+    "../node_modules/babylonjs-gui/babylon.gui.js.map")
+
+set(DEPENDENCIES
+    "../Dependencies/ammo.js"
+    "../Dependencies/meshwriter.min.js"
+    "../Dependencies/recast.js")
```

Comment:
nit: I would not bother creating a separate list of these. Conceptually, they are still babylon scripts. They just don't come from NPM.